src/sentry/api/serializers/rest_framework/base.py

```diff
@@ -26,7 +26,9 @@ def convert_dict_key_case(obj, converter):
     Recursively converts the keys of a dictionary using the provided converter
     param.
     """
-    if not isinstance(obj, dict):
+    if isinstance(obj, list):
```

Comment:
Can you add a test to cover this case in `CamelSnakeModelSerializerTest`? (or just add a test for `convert_dict_key_case`)

Reply:
Added a test :)